float64 time
uint16[] data
